components/body/desktop.styl

```diff
@@ -0,0 +1,116 @@
+@media (min-device-width 1200px)
+  .main-side-margin
+    margin 0 170px
+
+  .body-content-container
+    padding-top 0px
+
+  #section1.body-section-content, .section-three .body-section-content, .section-four .body-section-content, .body-section-five-content.js-section-content
```

Comment:
This looks like it could be abstracted a bit to `.flex-row` or something similar.